### DISCUSSION TITLE
제페토 월드 템플릿 인스톨러 설치오류

### DISCUSSION THREAD

**Discussion by HwangYunHee** [naverz/zepeto-studio-kor#869] [5. Q&A]
제페토 월드 템플릿 인스톨러 다운 받아서 설치하니 에러가 발생합니다.
ExdutionEngineException: String conversion error: Illegal byte sequence encounted in the input.

이런 에러가 발생하는데요..
유니티 프로젝트 버전은 이상이 없습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 설치한 컴퓨터의 개발 환경이 어떻게 되시나요?
원인 파악 및 해결에 도움을 드리기 위해서 아래 양식에 따라 상세하게 제보 부탁 드립니다.

https://github.com/naverz/zepeto-studio-kor/issues
